protocol/avdl/keybase1/notify_team.avdl

```diff
@@ -16,4 +16,5 @@ protocol NotifyTeam {
 
   void teamChanged(TeamID teamID, string teamName, Seqno latestSeqno, TeamChangeSet changes) oneway;
   void teamDeleted(TeamID teamID) oneway;
+  void teamExit(TeamID teamID) oneway;
```

Comment:
It seems this broke CI: https://circleci.com/gh/keybase/kbfs/4784

Is it OK to implement this as a no-op in KBFS client?